components/form/ProjectDescriptionPrivate.tsx

```diff
@@ -17,7 +18,17 @@ export default function ProjectDescription({
     formState: { errors },
   } = useFormContext();
 
-  const router = useRouter();
+  const [clustersList, setClustersList] = useState(clusters);
+
+  const { data: session } = useSession({
+    required: true,
+  });
+
+  useEffect(() => {
+    if (session && !session?.isAdmin) {
```

Comment:
I think `session &&` is redundant as we use `?.isAdmin`?

Reply:
removed, thanks